include/User.h

```diff
@@ -1,14 +1,33 @@
-//
-// Created by [] on []
-//
-
 #ifndef RA7AL_USER_H
 #define RA7AL_USER_H
 
+#include <vector>
+#include <string>
+#include "Itinerary.h"
+#include "PaymentMethod.h"
+#include "Budget.h"
+#include "Language.h"
+using namespace std;
 
-class User {
+class User
+{
+private:
+    string email;
+    string password;
+    vector<Itinerary> itineraries;
+    vector<PaymentMethod> payments;
+    Budget budget;
+    vector<string> preferences;
+    Language language;
```

Comment:
Most of these data members will be inherited from the Person class. Keep the unique ones only.